src/protocols/bitcoin/components/TransferSendForm.vue

```diff
@@ -0,0 +1,234 @@
+        <!--            @click="setMaxValue"-->
+        <!--          >-->
+        <!--            {{ $t('common.max') }}-->
+        <!--          </BtnPlain>-->
+        <!--        </template>-->
+      </TransferSendAmount>
+    </template>
+
+    <template
+      #extra
+    >
+      <DetailsItem
+        :label="$t('modals.send.transactionSpeed')"
+      >
+        <template #value>
+          <TransactionSpeedPicker
+            :fee-list="feeList"
+            @changeFee="(value) => setFee(value)"
+          />
+        </template>
+      </DetailsItem>
+    </template>
+  </TransferSendFormBase>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  nextTick,
+  PropType,
+  ref,
+  watch,
+} from 'vue';
+import { useStore } from 'vuex';
+import BigNumber from 'bignumber.js';
+import {
+  useAccounts,
+  useBalances,
+} from '@/composables';
+import type {
+  TransferFormModel,
+} from '@/types';
+import { BTC_SYMBOL } from '@/protocols/bitcoin/config';
+import { useTransferSendForm } from '@/composables/transferSendForm';
+import { PROTOCOL_BITCOIN } from '@/constants';
+
+import { INFO_BOX_TYPES } from '@/popup/components/InfoBox.vue';
+import DetailsItem from '@/popup/components/DetailsItem.vue';
+import TransferSendFormBase from '@/popup/components/TransferSendFormBase.vue';
+import TransferSendRecipient from '@/popup/components/TransferSend/TransferSendRecipient.vue';
+import TransferSendAmount from '@/popup/components/TransferSend/TransferSendAmount.vue';
+import TransactionSpeedPicker from '@/popup/components/TransactionSpeedPicker.vue';
+
+import EditIcon from '@/icons/pencil.svg?vue-component';
+import DeleteIcon from '@/icons/trash.svg?vue-component';
+import PlusCircleIcon from '@/icons/plus-circle-fill.svg?vue-component';
+
+export default defineComponent({
+  name: 'BtcTransferSendForm',
+  components: {
+    TransactionSpeedPicker,
+    DetailsItem,
+    TransferSendAmount,
+    TransferSendRecipient,
+    TransferSendFormBase,
+  },
+  model: {
+    prop: 'transferData',
+  },
+  props: {
+    transferData: { type: Object as PropType<TransferFormModel>, required: true },
+  },
+  setup(props, { emit }) {
+    const store = useStore();
+
+    const hasMultisigTokenWarning = ref(false);
+    const isUrlTippingEnabled = ref(false);
+
+    const { balance } = useBalances({ store });
+    const {
+      activeAccount,
+    } = useAccounts({ store });
+
+    const {
+      formModel,
+      errors,
+      hasError,
+      invoiceId,
+      invoiceContract,
+      clearPayload,
+      openScanQrModal,
+      handleAssetChange,
+    } = useTransferSendForm({
+      transferData: props.transferData,
+    });
+
+    // TODO - fee set proper values
+    const fee = ref(new BigNumber(0));
+    const feeList = [
+      { fee: new BigNumber(0.1), time: 25 },
+      { fee: new BigNumber(0.2), time: 600 },
+      { fee: new BigNumber(0.3), time: 3540 },
```

Comment:
Time growing should be reversed. :)

Reply:
Right 😄 Just put it as a placeholder here